src/os/bluestore/bluestore_types.h

```diff
@@ -200,7 +202,7 @@ struct bluestore_blob_t {
   uint8_t csum_chunk_order;        ///< csum block size is 1<<block_order bytes
 
   bluestore_extent_ref_map_t ref_map; ///< references (empty when in onode)
-  interval_set<uint32_t> unused;   ///< portion that has never been written to
+  std::bitset<32> unused;       ///< portion that has never been written to
```

Comment:
or since we need to know the unsigned equivalent, #define the type (uint16_t) and use sizeof()*8 here.  or whatever works.